docs/source/install.rst

```diff
@@ -12,19 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
```

Comment:
Sorry @mrocklin , was referring to this ":" here.

Reply:
Ah, great